availproject/docs
# Add Nominating Info

Hi

This PR covers the following
- Pre nominating info
- When to use pools vs stake direct
- How to stake direct on validator
- How to join a nomination pools
- Aimed at users that will stake AVL. So aimed for end users wishing to stake/delegate AVL.

Please assist to add to the menu, was unable to add to the menu. Tried editing _meta.json but didn't stick/work.

Outside of this pr I would recommend moving existing [nomination pools](https://docs.availproject.org/docs/end-user-guide/nomination-pools) to the validator section. The existing write up is how to create a pool, this should be done by validators or providers (wallets, exchanges) looking to pool stake together. Not end users

Hope this helps and contributes to Avail.

Thanks
